src/utils.tsx

```diff
@@ -291,19 +291,13 @@ export function debounce(fn: any, delay: number) {
   };
 }
 
-export function getServerByPriority(
-  singleServer: string | null | undefined,
-  servers: ServerEntry[] | null | undefined
-) {
+export function getServerByPriority(servers: ServerEntry[] | null | undefined) {
   // Return the server URL value, by priority:
   // - single server mode
   // - most recently used
   // - default
-  return (
-    singleServer ||
-    (servers && servers.length && servers[0].server) ||
-    DEFAULT_KINTO_SERVER
-  );
+  console.table({ SINGLE_SERVER });
```

Comment:
Should we log `servers` here too? (last used servers)

Reply:
🤦🏻 that was accidentally left in from `console` debugging. Unless you think that logging would actually be helpful, I was thinking I'd remove this rather than add more.